i-Code-Doc/scripts/finetune_rvlcdip.sh

```diff
@@ -1,8 +1,5 @@
-# Copyright (c) Microsoft Corporation.
-# Licensed under the MIT license.
-
-CUDA_VISIBLE_DEVICES=0,1,2,3 PYTHONPATH='.' python run_rvlcdip.py \
---data_dir "./dataset/rvl-cdip" \
+CUDA_VISIBLE_DEVICES=0 PYTHONPATH='.' python run_rvlcdip.py \
```

Comment:
seems all scripts are changed to single-node finetuning. Have you tested on multi-GPU finetuning?

Reply:
Yeah multi-GPU should work fine